docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -265,6 +265,77 @@ While these are out of scope for the present guide,
+   * - Race-essentialism
+     - The belief that racial groups form discrete genetic categories; that individuals of the same racial category are biogenetically similar; and that different races are fundamentally different. 
+   * - Race-based
+     - The use of race as a key marker in research or policy. This is built on a belief in race-essentialism. 
+   * - Race-conscious
+     - A new approach to how research can include race. Race-conscious research attempts to be aware of the historic roots and the pervasive, systemic nature of oppression, in touch with social reality, and open to the importance of race, while understanding that race itself is a social construct without biological significance. Instead, it frames discrepancies seen between races as a result of historical context or racism.
+   * - Social construct
+     - Any idea or concept that doesn’t have its basis in objective reality but is real because society believes it to be so. Some examples might include the value of money (paper is not inherently valuable, it is valuable because society has collectively agreed that it is), what is fashionable currently (skinny or wide-leg jeans are only better because we believe it to be so), and racial groups. 
+   * - Health disparities
+     - Differences in health outcomes, such as life expectancy, disease burden, or other adverse health conditions between different groups of people within a population. The term is often used to describe differences seen between racial or ethnic groups, but it can also be applied to differences observed across any individual characteristic including geographies, gender, sexual orientation or socioeconomic status.
+   * - Risk factor
+     - Attributes, characteristics, or exposures that increase the likelihood that a person develops a disease or health disorder. Risk factors are diverse and can span attributes such as diet, physical activity, alcohol use, genetics, age, and experiences of discrimination.
+   * - Risk marker
+     - Similar to a risk factor but not thought to be causally related to the development of a disease or health disorder. Race and its relationship to cardiovascular disease is an example. Some races have higher rates of cardiovascular disease, and while these might be caused through experiences of racism, race itself is not thought to be causally linked with CVD.
+   * - Nativity
+     - The general definition refers to the occasion of one’s birth. However, it is used here to mean the country someone was born in. 
+   * - Colonialism
+     - The policy or practice of acquiring full or partial political control over another country, occupying it with settlers, and exploiting it economically.
```

Comment:
```suggestion
     - The policy or practice of acquiring full or partial political control over another country, possibly occupying it with settlers, and exploiting it economically.
```

Not all colonialism is settler colonialism: https://link-springer-com.offcampus.lib.washington.edu/chapter/10.1057/978-1-349-95232-8_10